CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## v0.32.0 (May 18, 2023)
+
+IMPROVEMENTS:
```

Comment:
Feels weird having an improvement that no one using consul-template as a tool can use... but I guess it is better to include it than not. /shrug

Reply:
agreed! 🤷🏾‍♂️